app/assets/javascripts/libs/magna-charta.min.js

```diff
@@ -0,0 +1,4 @@
+/*! Magna Charta - v3.0.0 - 2012-12-04
```

Comment:
Sorry if I'm being obtuse here but; Is the plan for this to be maintained as part of this repo now and the Magna-charta repo to stop being used, or is the intention that changes will be made in the magna-charta repo and then this file will be updated?

Let's put this in a govuk-components directory in lib and call it govspeak-magna-charta if we're just having the minified version here.